Libraries/Text/Text/NSTextStorage+FontScaling.m

```diff
@@ -7,8 +7,6 @@
 
 #import "NSTextStorage+FontScaling.h"
 
-#import <React/RCTLog.h>
```

Comment:
are these changes necessary? given that they aren't to js, maybe we can leave this in if it does indeed improve the quality of error message in expo go

Reply:
my fault to wrongly pick the commit. reverted. thanks for the good catch